action.yml

```
@@ -27,6 +27,10 @@ inputs:
     description: "Filenames to ignore, separated by commas"
     required: false
     default: ""
+  trailer:
+    description: "Use a trailer of your choice in place of [skip ci] in first line of commit"
+    required: false
+    default: ""
```

suggestion: since `[skip ci]` isn't configurable, it's likely not beneficial to make the trailer text configurable, especially since there's only one trailer that GitHub Actions support anyway. Consider a boolean or option instead

```suggestion
  skipCiMessageLocation:
    description: |
      Determines how to prevent another GitHub Actions run when `bump-version` creates a commit
      for the version update. The following options are supported:
      - `subject` (default): adds `[skip ci]` suffix to the end of the commit's subject line.
      - `trailer`: adds a `skip-checks: true` trailer to the commit.
    required: false
    default: "subject"
```